scripts/Util.py

```diff
@@ -3478,9 +3478,7 @@ def getPluginEntryPoint(self, plugin, process, current):
             plugindir += os.sep
 
         return {
-            "IceSSL" : plugindir + "Ice.dll:ZeroC.Ice.SslPluginFactory",
-            "IceDiscovery" : plugindir + "Ice.dll:ZeroC.Ice.Discovery.PluginFactory",
-            "IceLocatorDiscovery" : plugindir + "Ice:ZeroC.Ice.LocatorDiscovery.PluginFactory"
+            "IceSSL" : plugindir + "Ice.dll:ZeroC.Ice.SslPluginFactory"
```

Comment:
Do we still need this too?

Reply:
I left it since I don't know.

Reply:
This method could just return None now since there's no more plugins in C# but we need to fix the callers to not setup the plugin property if this method returns None.